### PR TITLE
Branch two

### PR DESCRIPTION
**Pull request by dewired**
Added codes for the Java Tweeting project for review.

### CODE REVIEW

File: binarySearch/.idea/workspace.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
       <file pinned="false" current-in-tab="true">
         <entry file="file://$PROJECT_DIR$/src/binarySearch.java">
           <provider selected="true" editor-type-id="text-editor">
```
Comment (NarendraPrabhu) [inline binarySearch/.idea/workspace.xml]: Please get rid of all the files except src/* pom.xml. Use .gitignore for the same.

File: maxHeap/src/maxHeap.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
+public class maxHeap{
```
Comment (NarendraPrabhu) [inline maxHeap/src/maxHeap.java]: Class name should always start with Capital letter, Editor should have warned. Please set up your IDE for the same - this should be enough :-D -  https://google.github.io/styleguide/javaguide.html

File: maxHeap/src/maxHeap.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            System.out.print( heap[i] + " ");
+        }
+
+    }
+
+    public static void main(String[] args) {
+
+        maxHeap dsa = new maxHeap(10);
+
+        dsa.insert(1);
```
Comment (NarendraPrabhu) [inline maxHeap/src/maxHeap.java]: could have used a loop running 8 times instead, that reduces lines of code.

File: tweeter/src/main/java/Tweeter.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import twitter4j.*;
+
+//import java.io.IOException;
+import java.util.List;
+
+public class Tweeter {
+
+    private static Twitter tweeter = TwitterFactory.getSingleton();
```
Comment (NarendraPrabhu) [inline tweeter/src/main/java/Tweeter.java]: Change static access, unless necessary do not use static in java - https://dzone.com/articles/why-static-bad-and-how-avoid

File: tweeter/target/classes/twitter4j.properties [hunk 5]
```diff
@@ -0,0 +1,9 @@
+//debug=true
```
Comment (NarendraPrabhu) [inline tweeter/target/classes/twitter4j.properties]: These type of configuration files and credentials should NEVER EVER be added to the repository, especially repositories of this nature, now Anybody can post anything using your credentials.

File: maxHeap/src/maxHeap.java (hunk 2 above)
Comment (dewired) [reply maxHeap/src/maxHeap.java]: Will keep this in mind from now on. Thanks for the link.

File: maxHeap/src/maxHeap.java (hunk 3 above)
Comment (dewired) [reply maxHeap/src/maxHeap.java]: Entries could be some random values as well, so didn't use the loop. But will keep it mind.

File: binarySearch/.idea/workspace.xml (hunk 1 above)
Comment (dewired) [reply binarySearch/.idea/workspace.xml]: Did the changes in the tweeting project. Was able to hide the properties file having my credentials as well.

File: tweeter/src/main/java/Tweeter.java (hunk 4 above)
Comment (dewired) [reply tweeter/src/main/java/Tweeter.java]: Thanks a lot for the link.

**Pull request merged**